test/viewer.jl

```diff
@@ -96,6 +96,9 @@ end
 
     @test v.filename === "foo.png"
     img = Javis._pluto_viewer(vid, length(frames), objects)
+    objects = vid.objects
+    frames = Javis.preprocess_frames!(objects)
```

Comment:
is this needed due to this PR?

Reply:
No. This is to test if the pluto viewer generates frames correctly.

Reply:
I was wondering why it wasn't there before

Reply:
I removed the black lines. https://github.com/Wikunia/Javis.jl/blob/master/test/viewer.jl#L94-L95